pom.xml

```diff
@@ -155,6 +155,17 @@
             <version>3.0.3</version>
             <scope>provided</scope>
             <type>maven-plugin</type>
+            <exclusions>
+              <exclusion>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+              </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpclient</artifactId>
```

Comment:
Since `findbugs-maven-plugin` is using `provided` scope, the package you are adding back should also use `provided` scope to avoid introducing transitive dependencies to downstream projects.

An alternative way would be not adding the new package.

See https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html

Reply:
Thanks Ming! Will update the pom file.